src/Database/Driver/Sqlserver.php

```diff
@@ -151,6 +151,12 @@ public function connect(): bool
         if ($config['multiSubnetFailover'] !== null) {
             $dsn .= ";MultiSubnetFailover={$config['multiSubnetFailover']}";
         }
+        if ($config['encrypt'] !== null) {
```

Comment:
These must be added to `$_baseConfig` above.

Reply:
done